src/storage/mvcc/reader/scanner/mod.rs

```diff
@@ -388,7 +395,12 @@ where
     I: Iterator,
 {
     let seek_key = user_key.clone().append_ts(write_start_ts);
-    default_cursor.near_seek_for_prev(&seek_key, &mut statistics.data)?;
+    match statistics.load_data_hint() {
```

Comment:
The `over_seek_bound` is not counted in backward scan, the hint may not work when reverse loading data.

Reply:
updated